src/styles/index.scss

```diff
@@ -1,3 +1,90 @@
+@keyframes rotates {
+  from {
+    rotate: 0;
+  }
+
+  to {
+    rotate: 360deg;
```

Comment:
The correct property for rotation is `transform: rotate(...)`, not `rotate`. So, you should change `rotate: 360deg;` to `transform: rotate(360deg);`.